tests/v3/covid-19/api_historical.spec.js

```diff
@@ -148,6 +148,38 @@ describe('TESTING /v3/covid-19/historical', () => {
 			});
 	});
 
+	// Testing fake province added specifically for handling recovered aggregate for Canada
+	it('/v3/covid-19/historical', (done) => {
+		chai.request(app)
+			.get('/v3/covid-19/historical')
+			.end((err, res) => {
+				testBasicProperties(err, res, 200, 'array');
+				res.body.length.should.equal(271);
```

Comment:
don't these types of values, the source could add/remove countries to their collection at will and it will break this test. If you want to make sure the object is full, check >= 1

Reply:
Will checking `>=1` serve our purpose here? Essentially my intention was to check if the fake province has been removed from the list of countries returned. Though it looks off, I am thinking about something like:
```js
res.body.forEach(country => { country.province.should.not.be('recovered-aggregate')});
```
Still new to chai. I am not sure it is even valid to do something like that.

Reply:
you already check that here though: 
```js
			.get('/v3/covid-19/historical/canada/recovered-aggregate')
			.end((err, res) => {
				testBasicProperties(err, res, 404, 'object');
				res.body.should.have.property('message');
				done();
			});
```
Also yeah that is valid I believe.

Reply:
> I have failed to test on my end. Running `npm run test` always ends in an error. It failed even before adding this particular tests therefore I don't believe it is because of them. I have noticed it is failing here as well. Below is the error message I am getting.
> 
> ```js
>  1) "before all" hook in "{root}"
> 
>   0 passing (1m)
>   1 failing
> 
>   1) "before all" hook in "{root}":
>      Error: Timeout of 70000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
>       at listOnTimeout (node:internal/timers:555:17)
>       at processTimers (node:internal/timers:498:7)
> ```

@nibble0101 I pushed some code to your branch that removes some useless scrapers, basically they take too long and we don't even support the endpoints anymore. Testing should work now, but also to test your own we have a script for that:
`npm run test-single tests/v3/covid-19/api_historical.spec.js`

Reply:
> you already check that here though:
> 
> ```js
> 			.get('/v3/covid-19/historical/canada/recovered-aggregate')
> 			.end((err, res) => {
> 				testBasicProperties(err, res, 404, 'object');
> 				res.body.should.have.property('message');
> 				done();
> 			});
> ```
> 
> Also yeah that is valid I believe.

Essentially there are two endpoints where the fake data has to be removed. In the test below I am trying to ensure the fake province has been removed if data for all countries has been requested.
```js
.get('/v3/covid-19/historical')
			.end((err, res) => {
				testBasicProperties(err, res, 200, 'array');
				res.body.length.should.equal(271);
```
Well what do you suggest? Should I remove the test case?